platform/pc/service.c

```diff
@@ -10,6 +10,7 @@
 #include <page.h>
 #include <storage.h>
 #include <symtab.h>
+#include <unix.h>
```

Comment:
It's not critical, but it seems like this could be more of a generic kernel thing rather than unix-specific.